### PR TITLE
docs/about.md: update team members list

### LINKED ISSUE #723: Add new team members to about.html
1. Add new members to the list
2. Move inactive members to a _Past members_ section.
   Daniel Berzin Chua Yuan Siang: Project Member Jan 2018 - Aug 2018
   Jia Zhixin: Project Member May 2018 - Aug 2018
   Rachael Sim Hwee Ling: Project Member Jan 2018 - Dec 2018

### PR DESCRIPTION
**Pull request by yash-chowdhary**
• [X] Documentation update
Fixes #723

Updated the team member list. Added new members to the list and moved inactive members to a new _Past Members_ section.
Closed previous PR because I pushed package.lock accidentally.

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -14,13 +14,20 @@ The **project team**:
...
-* [**Rachael Sim Hwee Ling**](https://github.com/rachx): _Project Member_ since Jan 2018
+* [**Lin Si Jie**](https://www.github.com/sijie123): _Project Member_ since Jan 2019
+* [**Lu Yang Kenneth**](https://github.com/luyangkenneth): _Project Member_ since Jan 2019
+* [**Maddi Aadyaa**](https://www.github.com/amad-person): _Project Member_ since Jan 2019
+* [**Marvin Chin Kun Song**](https://www.github.com/marvinchin): _Project Member_ since Jan 2019
 * [**Tan Wang Leng**](https://github.com/yamgent): Current _Team Lead_ since Jul 2018, _Project Member_ for May 2018 - Jun 2018
+* [**Tan Zhen Yong**](https://www.github.com/Xenonym): _Project Member_ since Jan 2019
+
+Past Members:
+* [**Daniel Berzin Chua Yuan Siang**](https://github.com/danielbrzn): _Project Member_ Jan 2018 - Aug 2018
```
Comment (acjh) [inline docs/about.md]: Please help to standardize the `for` keyword before time ranges. Not sure if having or not is better. :see_no_evil:

Comment (yash-chowdhary) [reply docs/about.md]: Will do 👍

File: docs/about.md [hunk 2]
```diff
@@ -14,13 +14,20 @@ The **project team**:
 * [**Aaron Chong Jun Hao**](https://github.com/acjh): _Project Mentor_ since Jul 2018, _Team Lead_ for Aug 2017 - Jun 2018
 * [**Chng Zhi Xuan**](https://github.com/Chng-Zhi-Xuan): _Project Member_ since May 2018
 * [**Chua Yun Zhi Nicholas**](https://github.com/nicholaschuayunzhi): _Project Member_ since Jan 2018
-* [**Damith C. Rajapakse**](https://github.com/damithch): _Project Mentor_ since Aug 2016
-* [**Daniel Berzin Chua Yuan Siang**](https://github.com/danielbrzn): _Project Member_  Jan 2018 - Aug 2018
+* [**Damith C. Rajapakse**](https://www.comp.nus.edu.sg/~damithch/): _Project Mentor_ since Aug 2016
 * [**Jamos Tay**](https://github.com/jamos-tay): _Project Member_ since Aug 2018
-* [**Jia Zhixin**](https://github.com/nusjzx): _Project Member_ May 2018 - Aug 2018
 * [**Jiang Sheng**](https://github.com/Gisonrg): _Founding Member_ and _Team Lead_ for Aug 2016 - Jul 2017
```
Comment (yamgent) [inline docs/about.md]: @damithc should we update for Jiang Sheng as well?

Comment (damithc) [reply docs/about.md]: Yes, we can do that.

**Pull request merged**